### ISSUE TITLE
Update GeoNoteBook for GeoPySpark 2.0

### ISSUE THREAD

**Issue by jbouffard** [locationtech-labs/geopyspark#268]
Once `milestone/mvp` is merged, the [forked GeoTrellis GeoNoteBook](https://github.com/geotrellis/geonotebook) will not work. While there's one PR up right now to partially fix it [link](https://github.com/geotrellis/geonotebook/pull/7), more PRs will need to be made to get it in a working state.

This issue would best be solved once all of the PRs are merged into `milestone/mvp`.

**Comment by jbouffard** [comment]
Resoled by #483 #490 #467 #462 #460 #459 #457 #452
